recording.py

```diff
@@ -61,11 +63,16 @@ def run_session(trials_per_stim=3, trial_duration=1, trial_gap=1):
     raw = convert_to_mne(board.get_board_data())
     board.stop_stream()
     board.release_session()
-
     return raw
 
-def show_stim_progress(win,counter, total):
-    txt = visual.TextStim(win=win, text=f'{counter}/{total}', color=(0, 0, 0), bold=True, pos=(0, 0.8))
+def show_stim_progress(win,counter, total, stim):
+    if stim == 1:
```

Comment:
We have a class named `Marker` for mapping between stimuli and their integer representation. Import it like this: 

`From Marker import Marker`

and then, `Marker(1).name` will evaluate to `RIGHT`